pkg/scalers/nats_jetstream_scaler.go

```diff
@@ -33,6 +35,7 @@ type natsJetStreamScaler struct {
 
 type natsJetStreamMetadata struct {
 	monitoringEndpoint     string
+	useHTTPS               bool
```

Comment:
Private members use camel case
```suggestion
	useHttps               bool
```